scalardb-test/build.gradle

```diff
@@ -10,6 +10,13 @@ repositories {
         url 'https://oss.sonatype.org/content/repositories/snapshots/'
     }
     jcenter()
+    maven {
+        url = uri("https://maven.pkg.github.com/scalar-labs/scalardb-sql")
+        credentials {
+            username = project.findProperty("gprUsername") ?: System.getenv("GPR_USERNAME")
+            password = project.findProperty("gprPassword") ?: System.getenv("GPR_PASSWORD")
+        }
+    }
```

Comment:
After this change, we need the properties `gprUsername` and `gprPassword`, or the environment variables `GPR_USERNAME` and `GPR_PASSWORD` to build this scalardb-test.